lib/schemas/operationContext.js

```diff
@@ -2,9 +2,36 @@
  * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
-const sec = require('security-context');
+const {CONTEXT_URL: WEBKMS_CONTEXT_URL} = require('webkms-context');
 
 module.exports = {
-  type: 'string',
-  enum: [sec.constants.SECURITY_CONTEXT_V2_URL]
+  anyOf: [{
+    type: 'array',
+    items: [{
+      const: WEBKMS_CONTEXT_URL,
+    }, {
+      const: 'https://w3id.org/security/suites/ed25519-2020/v1',
+    }]
+  }, {
+    type: 'array',
+    items: [{
+      const: WEBKMS_CONTEXT_URL,
+    }, {
+      const: 'https://w3id.org/security/suites/aes-2019/v1',
+    }]
+  }, {
+    type: 'array',
+    items: [{
+      const: WEBKMS_CONTEXT_URL,
+    }, {
+      const: 'https://w3id.org/security/suites/hmac-2019/v1',
+    }]
+  }, {
+    type: 'array',
+    items: [{
+      const: WEBKMS_CONTEXT_URL,
+    }, {
+      const: 'https://w3id.org/security/suites/x25519-2020/v1',
+    }]
+  }]
```

Comment:
I don't think the above is what we want. We should not be accepting unnecessary suite contexts on non-GenerateKeyOperations. Only the gen key op needs a context with an array that includes the suite for the key that is being generated. The other ops should just use a string value for the context and it must be the WebKMS context.

Reply:
I see. so, make a separate schema just for the generate key operation (and use this array). And all the other operations just have the one context.

Reply:
@dmitrizagidulin Yeah, that's one option -- except that we could still wind up with some broken things. We don't want to accept an operation with a context of X for a key type of Y. So this array as it stands is just accepting *any* of the possible context combinations without linking it to the actual data (the `type` used in the data).